presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,50 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
+* Handle common disk failures during the spill. When on of disks got broken, it Presto will use another
+  one without failing the query. (:issue:`2444`)
+* Reduce number of metadata queries when querying ``information_schema``. (:issue:`2488`)
+* Limit broadcasted table size to 100MB by default when using the ``AUTOMATIC`` join reordering strategy. (:issue:`2527`)
+
+Security Changes
+----------------
+
+* Add ``ldap.bind-dn`` and ``ldap.bind-password`` LDAP properties to allow LDAP authentication
+  access LDAP server using service account. (:issue:`1917`)
+
+CLI Changes
+-----------
+
+* Allow inspecting network requests. (:issue:`2524`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect data returned when using S3 Select on uncompressed files. In our testing, S3 Select
+  was apparently returning incorrect results when reading uncompressed files, so S3 Select is disabled
+  for uncompressed files. (:issue:`2399`)
+* Fix incorrect data returned when using S3 Select on a table with ``skip.header.line.count`` or
+  ``skip.footer.line.count`` property. S3 Select API does not support skipping footers or more than one
+  line of a header.  In our testing, S3 Select was apparently sometimes returning incorrect results when
+  reading a compressed file with header skipping, so S3 Select is disabled when any of these table
+  properties is set to non-zero value. (:issue:`2399`)
+* Fix ``INSERT`` or ``CREATE TABLE ... AS`` query failure when one of the inserted ``REAL`` or ``DOUBLE`` values
+  is infinite or ``NaN``. (:issue:`2471`)
+* Add explicit support for conversion from Parquet decimal column type to Presto integer column types. (:issue:`2451`)
```

Comment:
```
* Support reading data when column is declared as an integer type on partition level but is a decimal type
  in Parquet file. (:issue:`2451`)
```

@sopel39 is this a regression fix?

Reply:
@findepi It wasn't explicitly supported (by coincidence really), but yes, it worked before.